packages/webapp/pages/posts/[id].tsx

```diff
@@ -129,7 +129,6 @@ const PostPage = ({ id, postData }: Props): ReactElement => {
         isLoading={isLoading || !isFetched}
         enableAuthorOnboarding={!!router.query?.author}
         enableShowShareNewComment={!!router?.query.new}
-        initializeNewComment={!!router?.query.c}
```

Comment:
Now that we won't be needing this anymore, especially removed in the `usePostComment`, I think we can also remove the one from the receiving end, which is I think in `PostContent`.

Reply:
My bad. It was already removed.